docs/moisture_dataset.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+import xarray
+
+def attribute_fix(ds):
+    """Needed to fix a low-level bug in ecCodes.
+    
+    Sometimes, shortNames get overloaded in ecCodes's table. 
+    To eliminate ambiguity in their string matching, we
+    force ecCodes to make use of the paramId, which is a
+    consistent source-of-truth.
+    """
+    for var in ds:
+        attrs = ds[var].attrs
+        result = attrs.pop('GRIB_cfName', None)
+        result = attrs.pop('GRIB_cfVarName', None)
+        result = attrs.pop('GRIB_shortName', None)
+        ds[var].attrs.update(attrs)
+    return ds
+
+
+def compute_gg_ll_dataset(ml_surface, ml_wind, ml_moisture, datestring):
+   """
```

Comment:
Comment should start on the same line as the open quotes. It should also begin with a 1 line description, e.g.
"""One line description.

<body>
"""

Reply:
Added the one-line description.